doc/import_samples/criteria.yml

```diff
@@ -1,41 +1,41 @@
 Quality of Writing:
     type: rubric
     max_mark: 24.0
-    level_0:
-        name: Beginner
+    Beginner:
```

Comment:
@YolandaZIK This is better, but not quite right: you should create a new key called `levels`, and nested all of this data under that key.

Reply:
(You did this correctly in the examples below, so I think you just forgot this one.)